x-pack/plugins/ml/public/application/explorer/actions/load_explorer_data.ts

```diff
@@ -214,6 +220,7 @@ const loadExplorerDataProvider = (
       tap(explorerService.setChartsDataLoading),
       mergeMap(
         ({
+          allAnnotations,
```

Comment:
Wondering if we should rename everything related to `allAnnotations` to `overallAnnotations`. "all" might be a bit misleading since `ANNOTATIONS_TABLE_DEFAULT_QUERY_SIZE = 500` limits how many annotations we fetch and it's not really "all".

Reply:
Updated here [`bc00e98` (#97202)](https://github.com/elastic/kibana/pull/97202/commits/bc00e98d645d385fb388eeb331469e3f89f51d5e)